### ISSUE TITLE
FLUX training all blocks

### ISSUE THREAD

**Issue by PluginBOXone** [kohya-ss/sd-scripts#2082]
What am I doing wrong? The script is training all blocks for FLUX.1 Dev, I defined this as my network args

--network_args train_double_block_indices=1 --network_args train_single_block_indices=1

so it should only train block one on each right? But the logs says different infos:

 INFO     create LoRA network. base dim      lora_flux.py:594
                             (rank): 64, alpha: 64.0
                    INFO     neuron dropout: p=None, rank       lora_flux.py:595
                             dropout: p=None, module dropout:
                             p=None
                    INFO     train all blocks only              lora_flux.py:605
                    INFO     create LoRA for Text Encoder 1:    lora_flux.py:741
                    INFO     create LoRA for Text Encoder 1: 72 lora_flux.py:744
                             modules.
2025-05-09 20:15:30 INFO     create LoRA for FLUX all blocks:   lora_flux.py:765
                             193 modules.
                    INFO     enable LoRA for text encoder: 72   lora_flux.py:911
                             modules
                    INFO     enable LoRA for U-Net: 193 modules lora_flux.py:916
FLUX: Gradient checkpointing enabled. CPU offload: False
prepare optimizer, data loader etc.
                    INFO     Text Encoder 1 (CLIP-L): 72       lora_flux.py:1018
                             modules, LR 5e-05

what am I doing wrong?
